contrib/http_util/src/main/java/io/opencensus/contrib/http/util/CloudTraceFormat.java

```diff
@@ -58,7 +58,7 @@
  *   <li>"105445aa7843bc8bf206b120001000/123;o=0"
  * </ul>
  */
-final class CloudTraceFormat extends TextFormat {
+public final class CloudTraceFormat extends TextFormat {
```

Comment:
why public?

Reply:
I am trying not to duplicate the doc in CloudTraceFormat. I will revert this change back and adopt the solution you mentioned below.